msrest/configuration.py

```diff
@@ -74,6 +74,11 @@ def __init__(self, base_url, filepath=None):
             requests.__version__,
             msrest_version)
 
+        # Users hooks. Must respect requests hook callback signature
```

Comment:
"User hooks"

End the sentences in a period to know it's finished (else it reads weird that suddenly the next line is capitalized).